### PR TITLE
Implemented RPM packaging + hosting

### PR DESCRIPTION
**Pull request by NNowakowski**
- Added specification for RPM package building
- Replaced System category with Settings/DesktopSettings to meet RPM requirements (as described in here: https://en.opensuse.org/openSUSE:Packaging_desktop_menu_categories#SUSE_categories_enforcement) and more accurately reflect what this software does

I've also gone ahead to host RPM packages of the current version 2.2.0 on OBS for Fedora (stable+rawhide) aswell as OpenSUSE (leap+tumbleweed): https://build.opensuse.org/package/show/home:NNowakowski/Komorebi-Fork

### CODE REVIEW

**Review by Ev1lbl0w — APPROVED** [review]
Looks good to me

**Comment by Ev1lbl0w** [comment]
Thank you for offering RPM support!

@Kangie the CI failed because of outdated build steps. There's isn't anything here that should fail, but could you take a look before merging, please?

**Comment by Kangie** [comment]
@Ev1lbl0w Sorry, been a bit busy lately. I'll have a look into the Travis-CI config and see why it's failing and get this merged in the next few days. :)

**Review by Kangie — APPROVED** [review]
Looks good! Travis failures look to have been due to a weird PPA thing. Probably just a brief outage, triggered the build again and everything was OK. FYI, anyone with write access to the repo should be able to do that if logged into Travis. 

I'll update the Travis config at some point, this one is the old repo's config with changes to get it working, but it's not pretty and I'm sure that we can drop some of the steps now (as we're not doing them to build manually...)

**Comment by Ev1lbl0w** [comment]
Cherry-picked for 2.2

**Pull request merged**
